From 33c0b895ebbbb24e43e4dc3ab0c0222bf6894816 Mon Sep 17 00:00:00 2001
From: Zakariyya Mughal <zaki.mughal@gmail.com>
Date: Thu, 15 Mar 2018 14:20:22 -0400
Subject: [PATCH 4/4] Include windows.h for non-GUI Windows build

This is so that the Windows API functions are available for command line
use.
---
 k2pdfopt.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/k2pdfopt.c b/k2pdfopt.c
index 914dc80..bcabfdb 100644
--- a/k2pdfopt.c
+++ b/k2pdfopt.c
@@ -75,10 +75,12 @@
 
 #include <k2pdfopt.h>
 
-#if (defined(HAVE_K2GUI) && (defined(WIN32) || defined(WIN64)))
+#if (defined(WIN32) || defined(WIN64))
 #include <windows.h>
+#if (defined(HAVE_K2GUI))
 static void k2pdfopt_launch_gui(K2PDFOPT_CONVERSION *k2conv,STRBUF *env,STRBUF *cmdline,int ascii);
 #endif
+#endif
 
 
 int main(int argc,char *argv[])
-- 
2.20.1

